test/CMakeLists.txt

```
@@ -21,6 +21,13 @@ set_target_properties(Catch2WithMain PROPERTIES EXPORT_COMPILE_COMMANDS OFF)
 get_target_property(CATCH2_INCLUDE_DIRS Catch2 INTERFACE_INCLUDE_DIRECTORIES)
 target_include_directories(Catch2 SYSTEM INTERFACE ${CATCH2_INCLUDE_DIRS})
 
+# enable precompiled headers
+if (SFML_ENABLE_PCH)
+    message(VERBOSE "enabling PCH for Catch2")
```

Is the message needed?

```suggestion
    message(VERBOSE "Enabling PCH for Catch2")
```

---

We have the same message for every other PCH, with lowercase "enabling". This message is consistent with the status quo.